gapic/src/main/com/google/gapid/LoadingScreen.java

```diff
@@ -46,12 +71,77 @@ public LoadingScreen(Composite parent, Theme theme) {
     titleLabel.setFont(theme.bigBoldFont());
     titleLabel.setLayoutData(new GridData(SWT.CENTER, SWT.TOP, true, false));
 
+    Label versionLabel = createLabel(container, "Version " + GAPID_VERSION.toFriendlyString());
+    versionLabel.setForeground(theme.welcomeVersionColor());
+    versionLabel.setLayoutData(new GridData(SWT.CENTER, SWT.TOP, true, false));
+
     statusLabel = createLabel(container, "Starting up...");
-    statusLabel.setLayoutData(new GridData(SWT.LEFT, SWT.TOP, true, false));
+    statusLabel.setLayoutData(new GridData(SWT.CENTER, SWT.TOP, true, false));
+
+    GridLayout gridLayout = Widgets.withMargin(new GridLayout(3, false), 15, 5);
+    optionsContainer = Widgets.createComposite(container, gridLayout);
```

Comment:
You no longer need the local `gridLayout` variable, also, if you static import the `Widgets` functions you can write this as:
```
optionsContainer = createComposite(container, withMargin(new GridLayout(3, false), 15, 5));
```